checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java

```diff
@@ -101,6 +113,75 @@ public InitializationStore getExceptionalExitStore(Tree tree) {
         return getFieldAccessFactory().getReturnStatementStores(methodTree);
     }
 
+    /**
+     * Returns the fields that are not yet initialized in a given store, taking into account the
+     * target checker.
+     *
+     * <p>A field is initialized if
+     *
+     * <ul>
+     *   <li>it is initialized in the initialization store, i.e., has been assigned.
+     *   <li>it has a non-top qualifier that does not have the meta-annotation {@link
```

Comment:
What does the "it has" mean here? Also the "its declared qualifier" below confused me.
Is one about the value in the store the other the declared field type?

Reply:
Clarified in [aed8e24](https://github.com/eisop/checker-framework/pull/621/commits/aed8e241bca2625501d1315ebbbb82b32c816dd4)